INSTALL.md

```diff
@@ -28,7 +28,7 @@ Unless Vorta is started with the `--foreground` option, it will minimize to the
 
 
 ## Install from Source
-The generic way is to install it as Python package using [PIP](https://pip.readthedocs.io/en/stable/installing/). First [install](https://borgbackup.readthedocs.io/en/stable/installation.html) Borg using the package of your distribution or via PyPI. The latter needs some additional [source packages](https://borgbackup.readthedocs.io/en/stable/installation.html#dependencies). Then install Vorta from PyPI. Your local Python version should be >= 3.6.
+The generic way is to install it as Python package using [PIP](https://pip.readthedocs.io/en/stable/installing/). First [install](https://borgbackup.readthedocs.io/en/stable/installation.html) Borg using the package of your distribution or via PyPI. The latter needs some additional [source packages](https://borgbackup.readthedocs.io/en/stable/installation.html#dependencies). Then install Vorta from PyPI. Your local Python version must be >= 3.6.
```

Comment:
The first sentence talk about "using PIP" (the command is lowercase, so "pip" might be less confusing).

The second sentence talks about "via PyPI" and means the same.

Guess both should say either "via pip" or "via PyPI" if they mean the same method.

Reply:
Good point.
Please just change the file, "allow edits from maintainers" is enabled ;-)

Reply:
Done